src/main/java/drivermanager/DriverFactory.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ *
+ * @author Jorge Rodrigo Cáceres Velasco
+ */
+package drivermanager;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+/**
+ * This class selects the type of driver.
+ */
+public final class DriverFactory {
+    private DriverFactory() {
+    }
+
+    /**
+     * Gets the type of driver.
+     *
+     * @param type to define which type will be returned.
+     * @return the driver type.
+     */
+    public static WebDriver getDriver(final DriverType type) {
+
+        WebDriver driver;
+
+        switch (type) {
```

Comment:
It looks like it's not open for extension. Maybe, you can use check DriverManagerType included in the Bonigarcia dependency

Reply:
Im working on a refactor using Factory pattern, but for this stage I wont be abel to complete the refactor